web/styles/dark_theme.css

```diff
@@ -85,14 +89,21 @@
         box-shadow: inset 0 1px 0 hsl(0deg 0% 0% / 20%);
     }
 
-    .enter_sends_choices,
     #user_enter_sends_label,
     #realm_enter_sends_label {
         & kbd {
-            background-color: hsl(211deg 29% 14%);
-            border-color: hsl(211deg 29% 14%);
-            text-shadow: none;
-            color: hsl(236deg 33% 90%);
+            color: var(--color-kbd-text);
+        }
+    }
```

Comment:
Does this remaining line still do something? It seems like the `app_components.css` value does ths same thing?

Reply:
@timabbott Yes, it does. In the light theme, `#realm_enter_sends_label kbd` and `#user_enter_sends_label kbd` override the `color` property of `kbd` in `app_components.css`. However, in the dark theme, both `kbd` elements and `#realm_enter_sends_label kbd` and `#user_enter_sends_label kbd` share the same color.

Would you like me to create a color variable, `--color-enter-sends-label-kbd`, that will have a different color for the light theme but match `--color-kbd-text` for the dark theme? It might seem a bit redundant, but it could help keep the code organized. Let me know what you think.

Reply:
@sanchi-t, if it's possible, a better approach would be to iron out the problem selectors, and ensure the correct color variable is assigned in both light and dark mode.

Reply:
I could not find a better way to fix this than adding creating a new variable. The first commit in #31657 fixes this (Thought it was a pretty minor problem so I did not create a new PR).